Formula/open-mesh.rb

```diff
@@ -25,6 +28,14 @@ def install
         args << "-DBUILD_APPS=OFF"
       end
 
+      if build.without? "python"
+        args << "-DOPENMESH_BUILD_PYTHON_BINDINGS=OFF"
+      else
+        args << "-DOPENMESH_BUILD_PYTHON_BINDINGS=ON"
+      end
```

Comment:
maybe there are neat freaks who do not use the binding and want to keep their site-package dir clean.